docs/ReleaseNotes.md

```diff
@@ -1,289 +1,401 @@
-
-### Fixes
-
-- Schema is properly chained/inherited during Element construction.
-- Vines properly inherit their parent's schema.
-- Button icons can now be occluded.
-- Double cursor in experience list is removed.
-- Hidden buttons no longer can activate with voice commands (an audible bloop is still heard currently however).
-- Leaked GameObjects are cleaned up when exiting edit mode.
-
-***
-
-## 0.48.0 - 7/6/2020
-
-### Improvements
-
-- Better support for eye gaze.
-- Float UI elements are more responsive.
-- Cleaner color scheme and fonts.
-- Expanded metrics support
-
-### Bug fixes
-- Experience loading screen might start behind user.
-- Some menus couldn't be closed.
-- Experiences sometimes didn't load if an asset failed to load.
-
-***
-
-## 0.47.0 - 4/30/2020
-
-### Features
-
-- Users can create a new element by dragging an asset from the library and dropping into the hierarchy.
-- New users have access to Chapters 3 & 4.
-
-### Improvements
-
-- The menu item Help > Support links to a new feedback page.
-
-### Fixes
-
-- Fixed an error that occasionally appears when an Undo action removes an element.
-
-***
-
-## 0.46.0 - 4/16/2020
-
-### New Features
-
-- New Scripting API for player scores (preview). Docs are here
-- SAI can display prompts with button responses (preview). Docs are here
-- Unity users can convert a scan mesh to a .obj.
-
-### Improvements
-
-- Renamed the Unity menu "Tools" to "Enklu".
-
-### Fixes
-
-- Fixed error where calling undo resulted in an error
-
-***
-
-## 0.45.0 - 4/2/2020
-
-### Features
-
-- Undo/Redo via Edit menu or hotkeys (ctrl-z/ctr-shift-z on windows, cmd-z/cmd-shift-z on mac)
-- New experiences start with 3-point lighting
-- New users get Chapters 1 & 2
-
-### Improvements
-
-- Scenes to build are defined in Build Settings, not hard coded.
-- Updated ace text editor
-- Order of elements at the same depth of the hierarchy is now stored and respected
-
-### Fixes
-
-- Fixed issue where dragging elements in the hierarchy sometimes failed
+---
+id: ReleaseNotes
+title: Release Notes
+---
+
+## 2020.11.0 - 11/24/2020
+
+### New Features
+
+#### CAD and Video Asset Support
+
+##### Uploading CAD Files
+Do you use Enklu Cloud for your rapid prototyping needs? Are you interested in editing your CAD files in the web editor, without the extra hassle of exporting them from Unity?
+
+Starting with 2020.11, Enklu Cloud now supports direct imports for the following CAD formats:
+- .asm 
+- .sldasm, .sldasm.1 (in a .zip file with any dependencies)
+- .prt, .prt.1
+- .sldprt, .sldprt.1
+- .dwg
+- .dxf
+- .fbx
+- .obj
+- .pxz (in a .zip file with any dependencies)
+- .gltf
+- .3dm
+- .stl 
+
+And this is only the beginning! We'll be continuing to update Enklu Cloud to accommodate more and more CAD formats as time goes on. So if you use a format that isn't represented above, please [contact us](/contact)!
+
+![CAD Assets](/img/product/releases/2020.11/CAD_asset_support.gif)

```

Comment:
ha, yeah, CAD support doesn't make for the sexiest gif ever, but it does the job.